website/routes.py

```diff
@@ -262,6 +262,12 @@ def make_url_map(app):
             json_renderer
         ),
         Rule(
+            '/help/',
+            'get',
+            website_views.redirect_help_to_faq,
```

Comment:
I'm not certain this is the right approach. It seems like it would be simpler to make the `/faq/` `Rule` also include `/help/`, and save the user a request. In any case, [the other `/help/` route](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/routes.py#L280) needs to be disabled.